### DISCUSSION TITLE
PSI table failed: astits: Table CRC32 61b0d6b5 != computed CRC32 943efa59

### DISCUSSION THREAD

**Discussion by adminint** [bluenviron/mediamtx#2115] [Questions]
Hi, udp source not worked with error
For, example:
paths:
  1hd:
    source: udp://238.122.8.1:1234

2023/07/28 09:20:46 INF [path 1hd] [udp source] ERR: astits: building new data failed: astits: parsing PSI data failed: astits: parsing PSI table failed: astits: Table CRC32 8562bc56 != computed CRC32 ce293612
2023/07/28 09:20:51 INF [path 1hd] [udp source] ready: 1 track (H264)
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x9ac4ce]

goroutine 29 [running]:
github.com/asticode/go-astits.(*PSIData).toData(0xc00057dbe8?, 0xc00022a150, 0x66)
	/go/pkg/mod/github.com/asticode/go-astits@v1.11.0/data_psi.go:465 +0x10e
github.com/asticode/go-astits.parseData({0xc0001a2500?, 0x1, 0x400066?}, 0xbc?, 0xce69c0?)
	/go/pkg/mod/github.com/asticode/go-astits@v1.11.0/data.go:82 +0x578
github.com/asticode/go-astits.(*packetAccumulator).add(0xc0003912f0, 0xc00022a150)
	/go/pkg/mod/github.com/asticode/go-astits@v1.11.0/packet_pool.go:51 +0x232
github.com/asticode/go-astits.(*packetPool).add(0xc0003960e0, 0xc00022a150)
	/go/pkg/mod/github.com/asticode/go-astits@v1.11.0/packet_pool.go:104 +0x1f5
github.com/asticode/go-astits.(*Demuxer).NextData(0xc0005212c0)
	/go/pkg/mod/github.com/asticode/go-astits@v1.11.0/demuxer.go:142 +0xae
github.com/bluenviron/mediamtx/internal/core.(*udpSource).run.func1.1({0xfc85e0, 0xc0001a2018}, 0xc0005180a8, 0xfbe200?)
	/s/internal/core/udp_source.go:333 +0xca5
github.com/bluenviron/mediamtx/internal/core.(*udpSource).run.func1()
	/s/internal/core/udp_source.go:363 +0x32
created by github.com/bluenviron/mediamtx/internal/core.(*udpSource).run
	/s/internal/core/udp_source.go:184 +0x40a

**Comment by aler9** [comment]
Please use issues and not discussions to report bugs.
